homework-5/home-work-5-1.py

```diff
@@ -0,0 +1,17 @@
+var = input("Уведіть текст:")
+var_list = list(var)
+for i in var_list:
+    if str.isdigit(i) == True:
+        i_N=  int(i)
```

Comment:
Pay attention to naming and spaces:
- variables should be named in snake case (lowercase). So here it is better to use `i_n` insead
- operators should be separated from the rest of the code by one space before and one space after:
```
i = 1
if i % 2:
```